src/main/kotlin/blackJack/domain/Card.kt

```diff
@@ -1,3 +1,5 @@
 package blackJack.domain
 
-data class Card(val name: String, val number: Int)
+data class Card(val shape: Shape, val denomination: Denomination) {
+    override fun toString() = shape.shape + denomination.symbol
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    override fun toString() = "${shape.shape}${denomination.symbol}"
```